### PR TITLE
Update descriptive information for the top level of the API Specification

### PR DESCRIPTION
**Pull request by jguertin**

### CODE REVIEW

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 1]
```diff
@@ -168,18 +174,13 @@ paths:
...
-            type: integer
-          in: query
-          name: version
-          description: The version of the record to retrieve.  If not specified the most recent version will be retrieved.
         - schema:
             type: string
           in: header
           name: Account-Id
-          description: The current account represented by the user
+          description: The account that the user is operating on behalf of
```
Comment (bryan-gilbert) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml]: Do you want to be consistent with periods at the end of sentences?

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Personal Property Registry API
   version: '1.0'
-  description: PPR API Specs
+  description: 'The Personal Property Register (PPR) is a centralized register where legal claims to personal property can be registered, and consumers can check if there is any debt or obligation attached to goods they may wish to buy. The PPR API provides a real-time, secure, direct link between business systems and the PPR. Most PPR transactions, including searches, registrations, renewals, amendments and discharges, directly from your own software, instead of using the PPR website.'
```
Comment (bryan-gilbert) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml]: Can/should we place this somewhere on the UI side?  I'm thinking in the main html file.

**Review by bryan-gilbert — APPROVED** [review]
Nice!
Trust you'll take appropriate action on my comments.

File: ppr-api/docs/api_contract/ppr-api.v1.yaml (hunk 1 above)
Comment (jguertin) [reply ppr-api/docs/api_contract/ppr-api.v1.yaml]: Fixed

File: ppr-api/docs/api_contract/ppr-api.v1.yaml (hunk 2 above)
Comment (jguertin) [reply ppr-api/docs/api_contract/ppr-api.v1.yaml]: Out of scope here, makes sense to me, but I would think someone is responsible for that content.

**Pull request merged**